src/javascripts/ng-admin/Crud/form/FormController.js

```diff
@@ -71,20 +72,18 @@ export default class FormController {
             .then(() => $translate('CREATION_SUCCESS'))
             .then(text => notification.log(text, { addnCls: 'humane-flatty-success' }))
             .catch(error => {
-                const errorMessage = this.config.getErrorMessageFor(this.view, error) || 'ERROR_MESSAGE';
+                // this is kept for backward compatibility with the entity.errorMessage() method 
+                const msg = this.config.getErrorMessageFor(this.view, error) || 'ERROR_MESSAGE';
+                error.data.message = msg;
+                
+                const errorMessage = this.HttpErrorService.handleError(error);
```

Comment:
So thats an entirely new test on FormController right? Could not see any existing ones?